src/kifshare/core.clj

```diff
@@ -103,6 +104,12 @@
     (GET "/:ticket-id" [ticket-id :as request]
          (resp/content-type (t-c/get-ticket ticket-id request) "text/html; charset=utf-8"))))
 
+(defn- anon-files-routes []
+  (context "/anon-files" []
+    (HEAD ":filepath{.*}" [filepath] (a-c/handle-head filepath))
+    (GET ":filepath{.*}" [filepath :as req] (a-c/handle-get filepath req))
+    (OPTIONS ":filepath{.*}" [filepath] (a-c/handle-options filepath))))
```

Comment:
`:filepath{.*}` means, take everything from the start (including the leading `/`) which matches `.*` (i.e. everything) as the `filepath` argument. Better than using `/*` like anon-files was.